### DISCUSSION TITLE
Convert summary result as Prometheus Metrics

### DISCUSSION THREAD

**Discussion by mhoshi-vm** [aquasecurity/trivy-operator#88] [Development]
We'd like to see the vulnerability reports as a metric.
Similar ideas from here.
https://github.com/devopstales/trivy-operator#usage

**Comment by chen-keinan** [comment]
@mhoshi-vm are you familiar with the [starboard-exporter](https://github.com/giantswarm/starboard-exporter/) project ?
it produces metrics from `VulnerabilityReport, ConfigAuditReport` CRDs , trivy-operator CRDs are similar to starboard CRDs.
looking forward it on our plans to adjust the metrics to be more trivy oriented #78
